recipe/meta.yaml

```diff
@@ -5,10 +5,14 @@ package:
 source:
   fn: Python-3.5.1.tgz
   url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
-  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  sha256: 687e067d9f391da645423c7eda8205bae9d35edc0c76ef5218dcbe4cc770d0d7
+  patches:
+      - omit-local-site-packages.patch  # [osx]
 
 build:
-  number: 0
+  number: 0  # [win]
+  number: 0  # [linux]
+  number: 1  # [osx]
```

Comment:
So, I know we are only changing OS X here, but I'm ok with using one build number to keep life simple.